cmd/astro-wasm/astro-wasm.go

```diff
@@ -129,7 +135,7 @@ func preprocessStyle(i int, style *astro.Node, transformOptions transform.Transf
 		return
 	}
 	attrs := wasm_utils.GetAttrs(style)
-	data, _ := wasm_utils.Await(transformOptions.PreprocessStyle.(js.Value).Invoke(style.FirstChild.Data, attrs))
+	data, _ := wasm_utils.Await(transformOptions.PreprocessStyle.(js.Value).Invoke(style.FirstChild.Data, attrs, transformOptions))
```

Comment:
is this needed? I think that ProcessStyle has access to all of these options on the JS side anyways. Is there something that it needs?

Reply:
I think so. This was to have the object containing the hash value (e.g. `astro-hash1d2c3b4a`).

Reply:
We talked about this, make sense.